ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -190,6 +391,7 @@ private string GetCursorName(Decl decl)
                             decl.Location.GetFileLocation(out var file, out var _, out var _, out var offset);
                             var fileName = Path.GetFileNameWithoutExtension(file.Name.ToString());
                             name = $"__Anonymous{decl.Type.KindSpelling}_{fileName}_{offset}";
+                            AddDiagnostic(DiagnosticLevel.Info, $"Anonymous declaration found in '{nameof(GetCursorName)}'. Falling back to '{name}'.'", decl);
```

Comment:
Things like anonymous declarations or unsupported calling conventions are just `info`, since there isn't much we can do for it. We can really only generate code to the best of our ability.